docs/cip/README.md

```diff
@@ -4,704 +4,2802 @@ Title: Ouroboros Leios - Greater transaction throughput
+full 90 days.
+
+### Incentives
+
+<a id="incentive-structure" href="#incentive-structure"></a>The Leios incentive
+structure builds upon the existing Ouroboros Praos reward mechanism while
+introducing new considerations for EB production, voting participation, and
+dual-block production scenarios. The design aims to maintain economic security
+while optimizing for throughput efficiency.
+
+**Reward Function Evolution**
+
+<a id="current-reward-function" href="#current-reward-function"></a>The current
+Praos reward function tracks the number of RBs created by each SPO compared to
+their expected production, using this performance metric (β) to determine
+rewards proportional to stake. This mechanism detects offline behavior and
+incentivizes consistent participation without directly measuring transaction
+inclusion - empty blocks count equally toward the performance factor.
+
+<a id="performance-based-rewards" href="#performance-based-rewards"></a>**Performance-Based
+Rewards**: Leios extends this approach by refining the performance calculation
+to account for block validity in enhanced throughput scenarios. When operating
+within the rolling window (certificate-active periods), RBs containing invalid
+transactions due to incomplete ledger state knowledge should not negatively
+impact the producer's performance factor.
+
+**Invalid Transaction Disincentives**: The protocol implements specific
+mechanisms to disincentivize RB producers from including invalid transactions
+while protecting honest participants.
+
+> [!Note]
+>
+> **Design Rationale**
+>
+> EBs are at least an order of magnitude larger than Praos blocks/RBs, making it
+> more common that an EB is served late to some parties. By naively waiting for
+> the EB before processing the relevant RB (and related chain), a **new source
+> of delays** would be introduced that may affect the security of Praos (as it
+> increases Δ for the relevant delivery assumption). By allowing transactions
+> with potential UTXO conflicts in RBs (while maintaining all other validation
+> guarantees), this threat is mitigated by enabling adoption of chains without
+> having the relevant EBs.
+
+Rather than counting all RBs toward the performance factor β (the fraction of
+all blocks within an epoch that a stake pool created), the enhanced reward
+calculation counts only RBs containing exclusively valid transactions. This
+modification ensures that:
+
+1. **Honest party protection**: SPOs operating with incomplete ledger state can
+   always produce valid empty RBs, maintaining their performance factor even
+   when lacking complete EB synchronization
+2. **Invalid transaction penalty**: RBs containing any invalid transactions are
+   excluded from the β calculation, directly reducing the producer's reward
+   proportional to their stake
+3. **Maintained incentive structure**: The existing reward mechanism remains
+   intact while adding precision to performance measurement
```

Comment:
The whole incentive section up to this point is outdated (it assumes there can be invalid transactions in blocks).